go/tests/endpoint/endpoint.go

```diff
@@ -197,6 +173,18 @@ func IsAvailable(programs ...string) bool {
 	return false
 }
 
+func IsSecure() bool {
```

Comment:
Why is this back?

Reply:
copy/paste error on my part. i thought it was weird that it was back :)